client/js/lounge.js

```diff
@@ -170,6 +170,10 @@ $(function() {
 		$("#sign-in").remove();
 
 		var id = data.active;
+
+		// Set it right away before triggering a click so that "open" event is not emitted
```

Comment:
It's actually quite silly this way. If you open the page and `#channel1` is loaded, switch to another channel and then reload the page, `#channel1` will display the previous unread counter as it won't be reset.